cmd/root.go

```diff
@@ -195,12 +195,8 @@ func AddCommands(cmd *cobra.Command) {
 		config.TortoiseBeacon.GracePeriodDuration, "Grace period duration in milliseconds")
 	cmd.PersistentFlags().DurationVar(&config.TortoiseBeacon.ProposalDuration, "tortoise-beacon-proposal-duration",
 		config.TortoiseBeacon.ProposalDuration, "Proposal duration in milliseconds")
-	cmd.PersistentFlags().DurationVar(&config.TortoiseBeacon.FirstVotingRoundDuration, "tortoise-beacon-first-voting-round-duration",
```

Comment:
I missed that you removed its usage in https://github.com/spacemeshos/go-spacemesh/pull/2605. However, we need to be able to set the first voting round longer than the other ones

Reply:
why?

Reply:
the code complexity and amount of messages is the same, i haven't checked the size but it is probably the same as well

Reply:
We send lists of valid and invalid proposals in the first round, and a bit vector with a difference in the following rounds, so the following rounds take less time. Therefore we can reduce the duration of following rounds

Reply:
but what is the difference is it like 10kb vs 100b or 10mb vs 100b? if it is the former then simplicity > micro optimizations

Reply:
added first round timeout